test/object-to-arraylist.js

```diff
@@ -0,0 +1,20 @@
+import test from 'ava'
+import {object-to-arraylist} from '../src'
```

Comment:
For this to work, you'll need to update the `src/index.js` file like the rest of the other modules.